set.mm

```diff
@@ -42487,6 +42463,74 @@ Equality axioms (and Extensionality) yield
   elelpwi $p |- ( ( A e. B /\ B e. ~P C ) -> A e. C ) $=
     ( cpw wcel elpwi sseld impcom ) BCDEZABEACEIBCABCFGH $.
 
+  ${
+    $d A x $.  $d B x $.
+    $( The powerclass preserves inclusion.  See ~ sspwb for the biconditional
+       version.  (Contributed by NM, 13-Oct-1996.)  Extract forward implication
+       of ~ sspwb since it requires fewer axioms.  (Revised by BJ,
+       13-Apr-2024.) $)
+    sspw $p |- ( A C_ B -> ~P A C_ ~P B ) $=
+      ( vx wss cpw cv wcel sstr2 com12 velpw 3imtr4g ssrdv ) ABDZCAEZBEZMCFZADZ
+      PBDZPNGPOGQMRPABHICAJCBJKL $.
+  $}
+
+  ${
+    sspwi.1 $e |- A C_ B $.
+    $( The powerclass preserves inclusion (inference form).  (Contributed by
+       BJ, 13-Apr-2024.) $)
+    sspwi $p |- ~P A C_ ~P B $=
+      ( wss cpw sspw ax-mp ) ABDAEBEDCABFG $.
+  $}
+
+  ${
+    sspwd.1 $e |- ( ph -> A C_ B ) $.
+    $( The powerclass preserves inclusion (deduction form).  (Contributed by
+       BJ, 13-Apr-2024.) $)
+    sspwd $p |- ( ph -> ~P A C_ ~P B ) $=
+      ( wss cpw sspw syl ) ABCEBFCFEDBCGH $.
+  $}
+
+  ${
+    $d x A $.  $d x B $.
+    $( Equality theorem for power class.  (Contributed by NM, 21-Jun-1993.) $)
+    pweq $p |- ( A = B -> ~P A = ~P B ) $=
+      ( vx wceq cv wss cab cpw sseq2 abbidv df-pw 3eqtr4g ) ABDZCEZAFZCGNBFZCGA
+      HBHMOPCABNIJCAKCBKL $.
+  $}
+
+  ${
+    $d x A $.  $d x B $.
+    $( Alternate proof of ~ pweq directly from the definition.  (Contributed by
+       NM, 21-Jun-1993.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    pweqALT $p |- ( A = B -> ~P A = ~P B ) $=
+      ( vx wceq cv wss cab cpw sseq2 abbidv df-pw 3eqtr4g ) ABDZCEZAFZCGNBFZCGA
+      HBHMOPCABNIJCAKCBKL $.
+  $}
```

Comment:
The proof is exactly the same as for pweq!?

Reply:
Good catch. Somehow, the new proof of `pweq` was not copied. I'll update it... and I'll also check the other changes I made. FYI:
```
$ git show e3b0b8dc5355964a30c3468353757e1e4733a3fd
[...]
+  $( Equality theorem for power class.  (Contributed by NM, 21-Jun-1993.)
+     (Proof shortened by BJ, 13-Apr-2024.) $)
+  pweq $p |- ( A = B -> ~P A = ~P B ) $=
+    ( wceq cpw wss eqimss sspw syl eqimss2 eqssd ) ABCZADZBDZKABELMEABFABGHKBAE
+    MLEBAIBAGHJ $.
+
+  ${
+    $d x A $.  $d x B $.
+    $( Alternate proof of ~ pweq directly from the definition.  (Contributed by
+       NM, 21-Jun-1993.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    pweqALT $p |- ( A = B -> ~P A = ~P B ) $=
+      ( vx wceq cv wss cab cpw sseq2 abbidv df-pw 3eqtr4g ) ABDZCEZAFZCGNBFZCGA
+      HBHMOPCABNIJCAKCBKL $.
```

Reply:
Proof from #3918 restored in #3944.